### ISSUE TITLE
Weird dots underneath 'LTS'

### ISSUE THREAD

**Issue by kasiag** [canonical-websites/www.ubuntu.com#3906]
see screenshot
![screen shot 2018-08-16 at 13 45 48](https://user-images.githubusercontent.com/4948563/44209212-b4f6c580-a15a-11e8-9ffc-80085b492be9.png)

**Comment by pmahnke** [comment]
that's an abbr tag... an abbreviation, if you hover, it explains what LTS means... I assume it is a vanilla issue if you don't like the design

**Comment by Caleb-Ellis** [comment]
@pmahnke Yeah in the PR (#3909) I've just thrown in Vanilla 1.8.0 styling for `abbr` to make it look nicer, but not remove it
